### PR TITLE
Minigun Change

### PR DESCRIPTION
**Pull request by Sibg**
## About The Pull Request
Changes the Mini-gun name & description. It also balances a few things.
## Why It's Good For The Game
Makes the Mini-gun more lore friendly and make its description sound better. It also balances a few things.
## Changelog
:cl:
tweak: Changes the mini-gun name & description.
balance: minigun stats, making it what it should be, etc
/:cl:

### CODE REVIEW

**Comment by XElectricX** [comment]
S p e e d m e r g e. Also the "tweak" tag is a lie, this is a balance change ree.

**Comment by TheOneAndMightyRed** [comment]
Is it just me or does the client crash whenever you shoot a wall with autofire on the minigun?
Client keeps crashing for some reason while server is doing fine.

**Comment by InterroLouis** [comment]
That wield delay's a bit low, isn't it? Are you sure you didn't mean to put 'seconds' after that number? Because as is, that will give it a 0.12 _second_ wield delay. Everything else seems fine to me here, except for that.

File: code/modules/projectiles/guns/specialist.dm [hunk 1]
```diff
@@ -1003,29 +1003,28 @@
...
+	aim_slowdown = 0.8
 	flags_gun_features = GUN_AUTO_EJECTOR|GUN_WIELDED_FIRING_ONLY|GUN_LOAD_INTO_CHAMBER|GUN_AMMO_COUNTER
-	gun_firemode_list = list(GUN_FIREMODE_BURSTFIRE, GUN_FIREMODE_AUTOMATIC, GUN_FIREMODE_AUTOBURST)
+	gun_firemode_list = list(GUN_FIREMODE_AUTOMATIC)
 	attachable_allowed = list(
 						/obj/item/attachable/flashlight)
 	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 19,"rail_x" = 10, "rail_y" = 21, "under_x" = 24, "under_y" = 14, "stock_x" = 24, "stock_y" = 12)
 
-	fire_delay = 2
+	fire_delay = 0.175
```
Comment (psykzz) [inline code/modules/projectiles/guns/specialist.dm:1015]: this is pretty low no?

Comment (psykzz) [inline code/modules/projectiles/guns/specialist.dm:1015]:
```suggestion
	fire_delay = 0.175 SECONDS
```

**Pull request merged**